src/main/scala/org/constellation/domain/cluster/ClusterStorageAlgebra.scala

```diff
@@ -24,4 +25,28 @@ trait ClusterStorageAlgebra[F[_]] {
 
   def setNodeState(id: Id, state: NodeState): F[Unit]
 
+  def getActivePeersIds(withSelfId: Boolean = false): F[Set[Id]]
+  def getActivePeers: F[Map[Id, PeerData]]
+  def getActiveLightPeersIds(withSelfId: Boolean = false): F[Set[Id]]
+  def getActiveLightPeers(): F[Map[Id, PeerData]]
+  def getActiveFullPeersIds(withSelfId: Boolean = false): F[Set[Id]]
+  def getActiveFullPeers(): F[Map[Id, PeerData]]
+  def setActiveFullPeers(peers: Set[Id]): F[Unit]
+  def isAnActivePeer: F[Boolean]
```

Comment:
There is `node` and `cluster`. Node storage is about self-knowledge. Cluster storage about the other peers.
Here, `isAnActivePeer` takes no parameter so I think it should belong to node storage, not cluster storage.

Reply:
Sure, it still can be moved to where it fits more.